### ISSUE TITLE
Foundry 10 & QE 1.1.4: Tiles are not being created

### ISSUE THREAD

**Issue by spetzel2020** [spetzel2020/quick-encounters#131]
1. Open a QE
2. Select a Tile
3. Click the [Add tokens/tiles] button on the QE
4. The QE is deleted from the Scene and added to the QE, but without an icon
5. Press [Run QE]; the tile is not added back to the Scene

**Comment by spetzel2020** [comment]
The Tile seems to appear in the QE, but the icon is showing as a broken link, and when you click [Run QE] you get:
```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'activate')
    at QuickEncounter.run (QuickEncounter.js:1210:39)
```

**Comment by spetzel2020** [comment]
Switched back to `canvas.tiles.activate()` in v10+
Fixed in 1.1.5c (not yet released)

**Comment by spetzel2020** [comment]
Also fixed in 1.1.5d: Tile image and source were not being displayed in QE dialog

**Comment by spetzel2020** [comment]
Released in 1.1.5
